Lib/site.py

```diff
@@ -108,7 +108,7 @@ def abs_paths():
             pass
         try:
             m.__cached__ = os.path.abspath(m.__cached__)
-        except (AttributeError, OSError):
+        except (AttributeError, OSError, TypeError):
```

Comment:
Doesn't line-107 need catching TypeError too?

Reply:
perhaps it should... However since __file__ seems to be set in most cases it wont cause a TypeError.